src/mjolnir/graphtilebuilder.cc

```diff
@@ -187,6 +187,45 @@ GraphTileBuilder::GraphTileBuilder(const std::string& tile_dir,
   lane_connectivity_builder_.reserve(n);
   std::copy(lane_connectivity_, lane_connectivity_ + n,
             std::back_inserter(lane_connectivity_builder_));
+
```

Comment:
Needed to append forward restriction https://github.com/valhalla/valhalla/pull/2766/files#r570901414